tools/topology/topology1/sof/pipe-host-volume-playback.m4

```diff
@@ -29,11 +29,14 @@ C_CONTROLMIXER(Playback Volume, PIPELINE_ID,
 # Volume configuration
 #
 
-W_VENDORTUPLES(playback_pga_tokens, sof_volume_tokens,
+define(DEF_PGA_TOKENS, concat(`pga_tokens_', PIPELINE_ID))
+define(DEF_PGA_CONF, concat(`pga_conf_', PIPELINE_ID))
```

Comment:
@singalsu is this fixing a problem with released topologies, or something you noticed in experiments?

I would like to make sure this change does not alter control names for existing topologies, this might have userspace impact w/ UCM.

Reply:
Without this change, changing HDA-generic playback ramp length and type was not possible because some other tokens with same name override it. I noticed when testing volume zero crossings optimize and ramp setting had no impact. Not only playback paths used the same name but also it was used in a capture pipeline.

Reply:
The 250 ms playback ramp time is a regression that is fixed by this patch.